### ISSUE TITLE
Possibly redesign website

### ISSUE THREAD

**Issue by PixelatedLagg** [RazorSh4rk/random-word-api#25]
Hey there,

The current website for this API is lacking in accessibility. The colour palette is kind of all over the place and I am unable to read the URLs of endpoints after I click them. I'm more than willing to help with this if you are interested. Also, thanks for creating such a valuable and free API :D

**Comment by RazorSh4rk** [comment]
Yes please im garbage at designing any sort of frontend lol, just go wild, make sure it doesnt load half of npm and works on mobile

**Comment by PixelatedLagg** [comment]
#26 - finished the website redesign
